templates/repo/issue/view_content/add_reaction.tmpl

```diff
@@ -1,13 +1,13 @@
 {{if .ctxData.IsSigned}}
 <div class="item action ui dropdown jump pointing top right select-reaction" data-action-url="{{.ActionURL}}">
-	<a class="add-reaction">
+	<a class="add-reaction muted">
 		{{svg "octicon-smiley"}}
 	</a>
 	<div class="menu">
-		<div class="header">{{.ctxData.locale.Tr "repo.pick_reaction"}}</div>
+		<div class="header reactions-header">{{.ctxData.locale.Tr "repo.pick_reaction"}}</div>
 		<div class="divider"></div>
 		{{range $value := AllowedReactions}}
-			<a class="item reaction" data-tooltip-content="{{$value}}" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
+			<a class="item reaction" title="{{$value}}" aria-label="{{$value}}" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
```

Comment:
Why did we suddenly replace the tooltip with a title?

Reply:
There was a issue with the tooltip getting stuck after modal dismissal, but I think we fixed that now with `hideOnClick`, so I guess I can revert.

Reply:
Restored `data-tooltip-content`.